app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class InvitationsController < ApplicationController
+  def create
```

Comment:
Muszę ogarnąć jakiś tutorial jak pisać service objecty bo wiem po co to, ale jeszcze nie wiem jak to napisać